src/components/DatePickerDocked/utils.ts

```diff
@@ -0,0 +1,114 @@
+import type { TextStyle, ViewStyle } from "react-native";
+import type { ComponentStylePropWithVariants } from "src/types";
+
+export const MONTHS_DATA = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"]
+
+type States = '';
+
+type CustomProps = {
+    roundness?: string;
+    headerBackgroundColor?: string;
+
+    container?: ViewStyle;
+};
+
+export const datePickerDockedStyles: ComponentStylePropWithVariants<TextStyle, States, CustomProps> = {
+    roundness: 'shapes.corner.extraSmall',
+    headerBackgroundColor: 'colors.surface',
+
+    container: {
+        minHeight: 360,
+        minWidth: 360,
+    },
+};
+
+type DatePickerHeaderCustomProps = {
+    datePickerHeader: ViewStyle;
+    buttonContainer: ViewStyle;
+    buttonWrapper: ViewStyle;
+    spacer: ViewStyle;
+    yearButtonStyle?: ViewStyle;
+    yearInnerStyle?: ViewStyle;
+    yearLabelStyle?: TextStyle & { typescale?: string };
+    daysWrapperStyle?: ViewStyle;
+};
+
+export const datePickerDockedHeaderStyles: ComponentStylePropWithVariants<
+    TextStyle,
+    '',
+    DatePickerHeaderCustomProps
+> = {
+    datePickerHeader: {
+        position: 'absolute',
+        top: 0,
+        right: 0,
+        left: 0,
+        zIndex: 10,
+    },
+    buttonContainer: {
+        height: 46,
+        marginTop: 'spacings.2',
+        marginBottom: 'spacings.2',
+        marginRight: 'spacings.4',
+        flexDirection: 'row',
+        alignItems: 'center',
+    },
+    buttonWrapper: {},
+    spacer: { flex: 1 },
+    yearLabelStyle: {
+        typescale: 'typescale.bodyMedium',
+        opacity: 0.7,
+    },
+    yearButtonStyle: {
+        alignSelf: 'flex-start',
+        marginLeft: 6,
+        borderRadius: 'shapes.corner.extraSmall' as unknown as number,
+    },
+    yearInnerStyle: {
+        paddingLeft: 'spacings.4',
+        flexDirection: 'row',
+        alignItems: 'center',
+        borderRadius: 'shapes.corner.extraSmall' as unknown as number,
+    },
+    daysWrapperStyle: {
+        marginHorizontal: 'spacings.3'
+    }
+};
+
+type DatePickerMonthPickerCustomProps = {
+    month?: TextStyle;
+    selectedMonth?: TextStyle;
+    monthButton?: ViewStyle;
+    monthInner?: ViewStyle;
+    selectedMonthInner?: ViewStyle;
+    monthLabel?: TextStyle;
+};
+
+export const datePickerMonthPickerStyles: ComponentStylePropWithVariants<
+    TextStyle,
+    '',
+    DatePickerMonthPickerCustomProps
+> = {
+    backgroundColor: 'colors.surface',
+
+    month: {
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'flex-start'
+    },
+    selectedMonth: { color: 'colors.onPrimary' },
```

Comment:
Incorrect color. Please check the m3 guidlines

Reply:
See my comment on the above suggestion @thetaungg

Reply:
Updated comments to the previous suggestion.
Improvement required.

Reply:
resolved @shashank1010 @thetaungg